app/lib/bowling.rb

```diff
@@ -0,0 +1,109 @@
+class Bowling
+  def initialize
+    # => poins contain array with puntuation in game
+    @points = Array.new(10) { nil }
+    
+    # => rounds capture pins downs in all shots, it is matrix 10X2
+    @rounds = Array.new(11) { Array.new(2) { nil } }
+
+    # => turn is a turn played
+    @turn = 0
+  end
+
+  def capture(x, y)
+    begin
+      x = Integer(x)
+      y = Integer(y)
+    rescue
+      return nil
+    end
+    
+
+    if x.negative? || y.negative? || x > 10 || x > 10
+      return nil
+    end
+
+    if (@turn!=10) && ((x + y).negative? || (x + y) > 10)
+      return nil
+    end
+
+    @rounds[@turn] = [x, y]
+  end
+
+  def calculate
+    if @turn < 10
+      @points[@turn] = @rounds[@turn][0]+@rounds[@turn][1]
+    end
+    if (@turn - 2) >= 0
+      if String(@rounds[@turn - 1] == [10, 0]) && String(@rounds[@turn-2] == [10, 0])
+        @points[@turn - 2] += @rounds[@turn][0]
+        @points[@turn - 1] += @rounds[@turn][0]
+      end
+    end
+    if (@turn - 1) >= 0
+      if String(@rounds[@turn - 1][0] + @rounds[@turn - 1][1]) == 10
+        @points[@turn - 1] += @rounds[@turn][0]
+        if @rounds[@turn - 1] == [10, 0]
+          @points[@turn - 1] += @rounds[@turn][1]
+        end
+      end
+      if @turn < 10
+        @points[@turn] += @points[@turn - 1]
+      end
+    end
+
+    @turn += 1
+    @points
+  end
+
+  def bowPrint
```

Comment:
los metodos se nombran utilizando snake_case, por ejemplo
```suggestion
  def bow_print
```